am-lib/src/main/java/uk/gov/hmcts/reform/amlib/enums/AccessType.java

```diff
@@ -1,6 +1,5 @@
 package uk.gov.hmcts.reform.amlib.enums;
 
 public enum AccessType {
```

Comment:
I though we agreed that that remains unchanged and new enumerated type will be named `AccessorType`.

Reply:
👍